test/others/test_utils.py

```diff
@@ -1245,6 +1246,21 @@ def greet2(name: str):
     assert greet2("John") == "Hello John"
 
 
+def test_secure_model_loading():
+    # setting the flag explicitly to zero
+    os.environ["TORCH_FORCE_WEIGHTS_ONLY_LOAD"] = "0"
+    env_val = os.getenv("TORCH_FORCE_WEIGHTS_ONLY_LOAD")
+    assert env_val == "0"
+
+    # now testing if just importing haystack is enough to enable secure loading of pytorch models
+    import importlib
+    import haystack
```

Comment:
We already import `haystack` at the top of the module, I think `reload` would do what you want even without this line. `importlib` can be safely imported at the top of the module along with the others I think

Reply:
haystack still had to be imported but moved importlib to the top. Kept the haystack local because we didn't need it for the other tests.